content/SCALE/SCALETutorials/TopToolbar/SettingUpSystemEmail.md

```diff
@@ -99,7 +99,7 @@ Click **Send Test Email** to verify you receive an email.
 
 ## Setting up the Email Alert Service
 
-The final step to setting up system email and have alerts sent to that system email is to configure the email alert.
+The system email account will be sent a system health email every night/morning, if it has been configured. You can also add/configure the Email Alert Service to send timely email warnings, when the system hits a specific state that is [listed in Alert Settings]({{< relref "scale/scaleuireference/toptoolbar/alerts/alertsettingsscreen.md" >}}), to the email specified in the alert service.
```

Comment:
I'm guessing Email Alert Service is something in the UI, and if it is use bold emphasis. Also, always check for the word "will" and reword to active voice present tense, like this suggestion below...
```suggestion
The system email account is sent a system health email every night/morning, if it is configured. You can also add/configure the **Email Alert Service** to send timely email warnings, when the system hits a specific state that is [listed in Alert Settings]({{< relref "scale/scaleuireference/toptoolbar/alerts/alertsettingsscreen.md" >}}), to the email specified in the alert service.
```